src/common/models/data-series/data-series.snapshots.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ExpressionJS } from "plywood";
+import { DataSeriesPercentOf } from "./data-series";
+
+export class DataSeriesExpressionSnapshots {
```

Comment:
I see Snapshot as some kind of value from object that we want to use in test. Here, it's result of "toExpression" from DataSeries. It's not a Fixture, because you can't do with this anything else than just comparing with some other value. 

That said, I'm not attached to name "Snapshot" and we can come up with better term.